cmd/verification_build.go

```diff
@@ -0,0 +1,396 @@
+package cmd
```

Comment:
Should this file be called `verification_builder.go`?

Reply:
renamed https://github.com/onflow/flow-go/pull/2308/commits/5f6b80addf2b674dbc0f6c6e9a41dc9a277fac26